dx_see: 0.0
dx_shot: 5
dy_see: 54.0
dy_shot: 5
headingangle_compass: 0.0
see_halt: 3.0
shot_halt: 5
